Odata-docs/client/v6/basic-crud-operations.md

```diff
@@ -20,7 +24,7 @@ var context = new DefaultContainer(new Uri("https://services.odata.org/v4/(S(lqb
 var people = context.People.Execute();
 ```
 
-The `Execute()` API call will return an `IEnumerable<Person>`.
+The `Execute()` API call will return an `IEnumerable<Person>`. To use the async api the call for `Execute` can be replaced with `ExecuteAsync` which returns a `Task<IEnumerable<Person>>`
```

Comment:
Instead of discussing the returned type in the docs it might be easier to just specify the type in the code snippet instead of using `var`

Reply:
Also if you could fix the following lines:
L60, L76: `UserName` --> `userName`
L67: Please be noted --> Please note